docs/src/main/asciidoc/security-oidc-code-flow-authentication-concept.adoc

```diff
@@ -1233,7 +1233,7 @@ It is important that this error endpoint is a public resource to avoid the user
 == References
 
 * xref:security-oidc-configuration-properties-reference.adoc[OIDC configuration properties]
-* xref:security-openid-connect-providers.adoc[OpenID Connect providers]
+* xref:security-openid-connect-providers.adoc[Configuring Well-Known OpenID Connect Providers]
```

Comment:
Just a small update to the capitalization, it should be [Configuring well-known OpenID Connect providers]